### ISSUE TITLE
Is there any way that i can hide my video to other user?

### ISSUE THREAD

**Issue by saikiranoptisol** [blackuy/react-native-twilio-video-webrtc#186]
I was trying to disable the video for the other user so that the other user can see my video but i can able to see his video. to do that is there any prop. 

Note: i have used enabled prop and set to false for localvideo. but its the actual requirement I need. 
 Kindly help

**Comment by slycoder** [comment]
Does setLocal{Audio/Video}Enabled not work?

**Comment by saikiranoptisol** [comment]
if i use setLocal enabled to false. i cant able to see the local video of mine but what i exactly need is the other user must not see my video (remote video to other). is there any option that i can acheive this. 
Ex: 1 user turns of the video 2 user must see some dummy image in the place of remote video

**Comment by saikiranoptisol** [comment]
@slycoder  can you respond on this

**Comment by slycoder** [comment]
I don't think this is supported right now.  I think the easiest way to add support for this would be to add react native methods that just call unpublish{Video/Audio}Track.  Is that something you'd be interested in helping out on?

**Comment by codemeall** [comment]
@saikiranoptisol did you achieve video pause?, just need to pause video and user shouldn't be able to see my video if I pause
Can any body help me with this ?

**Comment by RajeshStark** [comment]
> I was trying to disable the video for the other user so that the other user can see my video but i can able to see his video. to do that is there any prop.
> 
> Note: i have used enabled prop and set to false for localvideo. but its the actual requirement I need. Kindly help



> @saikiranoptisol did you achieve video pause?, just need to pause video and user shouldn't be able to see my video if I pause Can any body help me with this ?

hi did you find any solution for that?
